common/scala/src/main/scala/org/apache/openwhisk/core/entity/ExecManifest.scala

```diff
@@ -58,6 +58,20 @@ protected[core] object ExecManifest {
     mf
   }
 
+  /**
+   * Reads runtimes manifest from runtime string
+   *
+   * @param runtime
+   * @return the manifest if initialized successfully, or an failure
+   */
+  protected[core] def initialize(runtime: String): Try[Runtimes] = {
```

Comment:
why not use the method above and set the `manifestOverride` to `Some(...)` at the call site?

Reply:
Yes, you are right, updated accordingly.